workspace-manager-clienttests/src/main/java/scripts/testscripts/ServiceStatus.java

```diff
@@ -0,0 +1,24 @@
+package scripts.testscripts;
+
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.api.UnauthenticatedApi;
+import bio.terra.workspace.client.ApiClient;
+import bio.terra.workspace.model.SystemStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.utils.WorkspaceManagerServiceUtils;
+
+public class ServiceStatus extends TestScript {
+    private static final Logger logger = LoggerFactory.getLogger(ServiceStatus.class);
+
+    @Override
+    public void userJourney(TestUserSpecification testUser) throws Exception {
+        ApiClient apiClient = WorkspaceManagerServiceUtils.getClient(server);
+        UnauthenticatedApi unauthenticatedApi = new UnauthenticatedApi(apiClient);
+        SystemStatus systemStatus = unauthenticatedApi.serviceStatus();
```

Comment:
I think this block does not do what we want it to do.

`WorkspaceManagerServiceUtils.getClient(server)` returns an `ApiClient` **with an access token set**, where the access token belongs to the `testRunnerServiceAccount`. It then uses this `ApiClient` to make an unauthenticated request.

Instead, I think this block should be generating an `ApiClient` **without** an access token set, so it is truly making an authenticated request.

what I'm not sure about: does `UnauthenticatedApi` discard the access token somewhere within itself? Even if it does, it doesn't feel good relying on that, we shouldn't set an access token at all. Should this be using `WorkspaceManagerServiceUtils.getClientForTestUser(null, server)` instead? And if so, can `WorkspaceManagerServiceUtils.getClient` be deleted entirely since it isn't used?

Reply:
Agreed that this should be calling WorkspaceManagerServiceUtils.getClient(null, server) instead, or some convenience wrapper (e.g. WorkspaceManagerServiceUtils.getClientWithoutAccessToken) for that.

Reply:
Refactored build client code.